src/main/java/uk/gov/hmcts/divorce/citizen/notification/SaveAndSignOutNotificationHandler.java

```diff
@@ -18,14 +17,12 @@
     @Autowired
     private CommonContent commonContent;
 
-    public void notifyApplicant(CaseData caseData, UserDetails user) {
-        final var invite = caseData.getCaseInvite();
-        final var isTriggeredByApp2 = invite != null && user.getId().equals(invite.getApplicant2UserId());
-        final var self = isTriggeredByApp2 ? caseData.getApplicant2() : caseData.getApplicant1();
-        final var partner = isTriggeredByApp2 ? caseData.getApplicant1() : caseData.getApplicant2();
+    public void notifyApplicant(CaseData caseData, boolean isApplicant1) {
+        final var self = isApplicant1 ? caseData.getApplicant1() : caseData.getApplicant2();
+        final var partner = isApplicant1 ? caseData.getApplicant2() : caseData.getApplicant1();
 
         notificationService.sendEmail(
-            user.getEmail(),
+            isApplicant1 ? caseData.getApplicant1().getEmail() : caseData.getApplicant2EmailAddress(),
```

Comment:
nah that'll need to go back - `getApplicant2EmailAddress` is not necessary what app logs in with. It's the invite email